### DISCUSSION TITLE
Unable to create topic without partition when I set ns-isolation-policy

### DISCUSSION THREAD

**Discussion by tianshimoyi** [apache/pulsar#18286] [Q&A] [answered]
![截屏2022-11-01 下午9 31 30](https://user-images.githubusercontent.com/68532999/199245228-1326cce8-19fc-41e4-ad4f-0a6eb051d4d7.png)

**Comment by Demogorgon314** [comment]
@tianshimoyi Can you share the reproduction step? What is the `ns-isolation-policy` value you set?

**Comment by tianshimoyi** [comment]
@Demogorgon314 The picture below is the details of the ns-isolation-policy I set, including all brokers. By the way, my pulsar is deployed via pulsar-helm-char@4.9.4 and the version is 4.9.3
<img width="1432" alt="截屏2022-11-02 上午10 01 26" src="https://user-images.githubusercontent.com/68532999/199377494-1632733f-bdba-49a3-b1e9-172daced2cee.png">

**Comment by tianshimoyi** [comment, marked as answer]
@Demogorgon314 It is a problem with the regular expression of the --primary parameter. I can modify the regular expression.
```yaml
--primary pulsar-huhao-broker-0.*,pulsar-huhao-broker-1.*,pulsar-huhao-broker-2.*
```

**Reply by Demogorgon314** [reply]
Yes, you can modify it to `pulsar-huhao-broker.*` and try again.

**Comment by tianshimoyi** [comment]
@Demogorgon314 it worked, thanks a lot
